suite/tests/api/ir_x86_0args.h

```diff
@@ -1,5 +1,5 @@
 /* **********************************************************
- * Copyright (c) 2011-2016 Google, Inc.  All rights reserved.
+ * Copyright (c) 2011-2022 Google, Inc.  All rights reserved.
```

Comment:
Just a header update?

Reply:
Ah, I changed this file to mark them X64_ONLY, but in fact they are valid in 32-bit mode on AMD processors, so I reverted those changes but failed to revert the date.  Reverting it now.